src/main/java/Duke.java

```diff
@@ -1,10 +1,241 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    /** count tracker for number of tasks in array */
+    private static int taskCount = 0;
+
+    /** count tracker for number of uncompleted tasks in array */
+    private static int numOfUncompletedTasks = 0;
+
+    /** Array of tasks */
+    private static final Task[] tasks = new Task[100];
+
+    public static void greeting() {
+        System.out.println("_____________________________________________________");
+        System.out.println("Hello! I'm Duke the Dancing Dragon.");
+        System.out.println("Anything I can help you with, young Padawan?");
+        System.out.println("_____________________________________________________");
+    }
+
+    public static void goodbyeMessage() {
+        System.out.println("It's over Anakin... I can finally eat my lun-");
+        System.out.println("_____________________________________________________");
+    }
+
+    public static void printDone(Task arg) {
+        System.out.println("You have marked item " + arg.description + " as done:");
+        System.out.println(arg.getStatusIcon() + " " + arg.description);
+    }
+
+    /**
+     * Returns the list number which is assigned to the task.
+     *
+     * @param arg user input that contains word [done].
+     * @return task number.
+     */
+    public static int getTaskNum(String arg) {
+        String[] words = arg.trim().split("[\\s]+");
```

Comment:
I feel like the [\\s]+ is somewhat of a magic literal; I think it would be good to assign it to a aptly named variable to make your code more readable